src/main/java/com/softserve/academy/spaced/repetition/logger/logs/log.txt

```diff
@@ -0,0 +1,4 @@
+2017-09-28 22:17:28,661 [WARN ][    logger.Log4jImpl] - Exception in service = com.softserve.academy.spaced.repetition.service.ImageService@76837082, method = Image com.softserve.academy.spaced.repetition.service.ImageService.addImageToDB(MultipartFile)
+2017-09-28 22:17:28,663 [WARN ][    logger.Log4jImpl] - Exception in controller = com.softserve.academy.spaced.repetition.controller.ImageController@22d9f079, method = ResponseEntity com.softserve.academy.spaced.repetition.controller.ImageController.addImageToDB(MultipartFile)
+2017-09-30 21:42:24,990 [WARN ][    logger.Log4jImpl] - Exception in service = com.softserve.academy.spaced.repetition.service.ImageService@1f0a62b3, method = Image com.softserve.academy.spaced.repetition.service.ImageService.addImageToDB(MultipartFile)
+2017-09-30 21:42:24,991 [WARN ][    logger.Log4jImpl] - Exception in controller = com.softserve.academy.spaced.repetition.controller.ImageController@4284caa4, method = ResponseEntity com.softserve.academy.spaced.repetition.controller.ImageController.addImageToDB(MultipartFile)
```

Comment:
I suppose as soon as you delete this file it'll be okay. Try just reverting changes.